cpp/include/cudf/rolling/range_window_bounds.hpp

```diff
@@ -26,15 +26,27 @@ namespace cudf {
  *
  * Similar to `window_bounds` in `grouped_rolling_window()`, `range_window_bounds`
  * represents window boundaries for use with `grouped_range_rolling_window()`.
- * A window may be specified as either of the following:
+ * A window may be specified as one of the following:
  *   1. A fixed-width numeric scalar value. E.g.
  *      a) A `DURATION_DAYS` scalar, for use with a `TIMESTAMP_DAYS` orderby column
  *      b) An `INT32` scalar, for use with an `INT32` orderby column
  *   2. "unbounded", indicating that the bounds stretch to the first/last
  *      row in the group.
+ *   3. "current row", indicating that the bounds end at the first/last
+ *      row in the group that match the value of the current row.
  */
 struct range_window_bounds {
  public:
+  /**
+   * The type of range_window_bounds.
+   */
+  enum class extent : int32_t {
+    CURRENT_ROW = 0,  /// Bounds defined as the first/last row that matches the current row.
+    BOUNDED,          /// Bounds defined as the first/last row that falls within
+                      /// a specified range from the current row
```

Comment:
```suggestion
                      /// a specified range from the current row.
```

Reply:
I just noticed that yesterday. Thanks for *point*-ing that out.

Reply:
Fixed.